src/components/single/CameraSwitcher.vue

```diff
@@ -441,6 +441,8 @@ export default defineComponent({
         });
         }, 1000);
       }
+
+      document.getElementById('btnUpdateELProperties')!.click()
```

Comment:
@justdecodeme  This is not a good practice. Because you already have the get-property function immediately before this line, why not utilise it. You should push the necessary EPCs to the el.opc.ops[0[.epc array. [Of course, without modifying the epcList itself.]

Reply:
Fixed